src/bsoncxx/lib/bsoncxx/v_noabi/bsoncxx/json.cpp

```diff
@@ -99,7 +99,7 @@ document::value BSONCXX_CALL from_json(stdx::string_view json) {
     return document::value{buf, length, bson_free_deleter};
 }
 
-document::value BSONCXX_CALL operator""_bson(const char* str, size_t len) {
+document::value BSONCXX_CALL operator"" _bson(const char* str, size_t len) {
```

Comment:
> C++11 [user-defined literals](https://timsong-cpp.github.io/cppwp/n3337/over.literal) note a space is required:

[CWG 1473](https://cplusplus.github.io/CWG/issues/1473.html) was accepted as a defect report and retroactively applied to C++11 in April 2013:

![image](https://github.com/user-attachments/assets/10e30890-f986-413b-8874-7d7a852e2fd9)

https://github.com/mongodb/mongo-cxx-driver/pull/1178 neglected to account for older compiler releases which might not have implemented the DR.

From [CWG 2521](https://cplusplus.github.io/CWG/issues/2521.html):

> CWG feels that the ostensible significance of whitespace in this context is unfortunate.

I guess we'll have to deal with the `Wdeprecated-literal-operator` warnings in the interim.

Reply:
Thank you for the references. TIL about the "Defect reports" section on cppreference.